database/insertData/laos/02_template_orgRegistration.js

```diff
@@ -0,0 +1,1003 @@
+                    children: ["applicationData.responses.expList", ""]
+                  }
+                }
+              }
+              {
+                actionCode: "joinUserOrg"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 102
+                # TO-DO -- update condition to just check Outcome
+                # (from applicationData)
+                condition: {
+                  operator: "="
+                  children: [
+                    {
+                      operator: "objectProperties"
+                      children: [
+                        "applicationData.outcome"
+                      ]
+                    }
+                    "APPROVED"
+                  ]
+                }
+                parameterQueries: {
+                  user_id: {
+                    operator: "objectProperties"
+                    children: ["applicationData.userId"]
+                  }
+                  organisation_id: {
+                    operator: "objectProperties"
+                    children: ["outputCumulative.organisation.id"]
+                  }
+                  user_role: "Owner"
+                }
+              }
+              {
+                actionCode: "grantPermissions"
+                trigger: ON_REVIEW_SUBMIT
+                sequence: 103
+                condition: {
+                  operator: "="
+                  children: [
+                    {
+                      operator: "objectProperties"
+                      children: [
+                        "applicationData.outcome"
+                      ]
+                    }
+                    "APPROVED"
+                  ]
+                }
+                parameterQueries: {
+                  username: {
+                    operator: "objectProperties"
+                    children: ["applicationData.username"]
+                  }
+                  orgName: {
+                    operator: "objectProperties"
+                    children: ["outputCumulative.organisation.name"]
+                  }
+                  permissionNames: ["reviewJoinOrg", "applyOrgLicense"]
+                }
+              }
+            ]
+          }
+          templatePermissionsUsingId: {
+            create: [
+              # Apply General
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "applyGeneral" }
+                }
+              }
+              # Apply OrgRegistration
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "applyOrgRego" }
+                }
+              }
+              # Review Org rego
+              {
+                permissionNameToPermissionNameId: {
+                  connectByName: { name: "reviewOrgRego" }
+                }
+                stageNumber: 1
+                levelNumber: 1
+                canSelfAssign: true
+              }
```

Comment:
Is the company that you can review been restricted already?

Reply:
I don't think it has yet, we haven't added any more complex restrictions in policies for reviews yet.